clicks.view.lkml

```diff
@@ -1,86 +1,118 @@
+# Details on the derived.clicks table is in the Snowplow Web Model documentation:
+# https://github.com/snowplow-proservices/ca.bc.gov-snowplow-pipeline/blob/master/jobs/webmodel/README.md#4-clicks
+
 view: clicks {
   sql_table_name: derived.clicks ;;
 
-
+  # duplicate
   dimension: p_key {
+    description: "The primary key, which is one of: User ID, Domain User ID, Session ID, or Click ID."
     primary_key: yes
     sql: ${user_id} || ${domain_userid} || ${session_id} || ${click_id} ;;
   }
 
   # DIMENSIONS
 
   # Application
+  # This section contains fields that are duplicated across over view files in this project
 
   dimension: app_id {
+    description: "The application idientifier."
     type: string
     sql: ${TABLE}.app_id ;;
     group_label: "Application"
   }
 
   # this is only available for WebTrends data
   dimension: dcs_id {
+    description: "The identifier generated by a hit, a Webtrends identifier."
```

Comment:
This is an id used to identify a profile in webtrends. Not really generated by a hit. But I don't think that is important. I maybe wrong but will this only be used by us for our testing? shouldn't this be hidden?

Reply:
You're right, I minterperated the line from the Webtrends documentation at https://help.webtrends.com/legacy/en/QueryParameters/dcsid.html stating "This parameter contains the value of the DCSID that generated the hit". I'll update to inidicate that DCS ID is a profile in WT.

I will also hide it since it is not useful to most users; and developers can unhide it.